src/components/Chatting/Header/Popup/PopupCancel.jsx

```diff
@@ -40,11 +40,10 @@ const PopupCancel = (props) => {
       roomId: props.roomId,
     });
     if (res.status === 200) {
-      axios.get("/rooms/v2/searchByUser").then(({ data }) => {
+      await axios.get("/rooms/v2/searchByUser").then(({ data }) => {
         setMyRoom(data);
       });
-      props.recallEvent();
-      history.push("/myroom");
+      history.replace("/myroom");
```

Comment:
```suggestion
const { data } = await axios.get("/rooms/v2/searchByUser");
setMyRoom(data);
history.replace("/myroom");
```

then을 사용하지 않아도 될 것 같습니다!